website/content/docs/k8s/connect/terminating-gateways.mdx

```diff
@@ -88,6 +88,57 @@ Registering the external services with Consul is a multi-step process:
 
 ### Register external services with Consul
 
+There are two ways to register an external service with Consul:
+1. If you have [`TransparentProxy`](/docs/k8s/helm#v-connectinject-transparentproxy) enabled, you can declare external endpoints with the [Destination](/docs/connect/config-entries/service-defaults#terminating-gateway-destination) field of `service-defaults`.
+1. You can add the service as a Node in the Consul catalog.
+
+#### Register an external service as a Destination
+
+Destinations are valid only for [`TransparentProxy`](/docs/k8s/helm#v-connectinject-transparentproxy) mode and allow clients to dial the external service directly.
+Also note these additional limitations when using Destinations to route traffic through a terminating gateway:
+
+| Layer 4 External Services (TCP) | Client dials hostname | Client dials IP |
+|---|---|---|
+| **with TLS** | ✅ Clients can dial successfully. `CAFiles` are not allowed because traffic is already end-to-end encrypted by the client. | ✅ Clients can dial successfully. CAFiles are not allowed because traffic is already end-to-end encrypted by the client. |
```

Comment:
A few things:
1. As far as I can see, we don't use the checkmark or cross-out icons anywhere else. For consistency, I think we should remove them.
2. It took me a while to figure out that these table represents a matrix with conditions along the x and y axises, as opposed to tabular reference information where the column headings represent a features associated with the items in the left-most column.
3. We introduce the matrix as describing limitations, but most of the reference points show successful calls. I would expect the matrix that describes limitations to only show instances when something is unsuccessful.

I think I would put this information into a more linear format.

Reply:
I was using the table [here](https://learn.hashicorp.com/tutorials/consul/terminating-gateways-connect-external-services#prerequisites) as prior art, although now I see they use the 🚫 symbols instead. I'm not married to this but I would like your take on the reference provided.

When you say linear format, would you be OK with something like the following that doesn't use the Y axis? I like some sort of table organization because I personally found it hard to grok all these possible scenarios without having them laid out together.

| Service Layer | TLS | Client dials | Works out of the Box | Notes |
|---|---|---|---|---|
| L4 | No | Hostname | 🚫 | Because the Sidecar is not protocol-aware, it will not be able to identify traffic going to the external service. |

Reply:
See changes for the actual layout I picked.